### PR TITLE
Avoid exporting environment variables

### LINKED ISSUE #10: Security: when run locally, plugin sends all env variables
The plugin sets all of `process.env` variables as secrets:
https://github.com/nimbella/netlify-plugin-nimbella/blob/108706af452c6f0c9b8675113ea5446756bc4fd6/index.js#L119

This can leak unwanted variables, especially when run locally. 

To reproduce run `netlify build` locally with the plugin configured and see the generated `env.json` file

> You can also run `nim action get <action>` to see all the env vars added during build time

### PR DESCRIPTION
**Pull request by satyarohith**
Closes #10

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,20 +27,14 @@ async function deployProject(run) {
...
-}) {
+async function deployActions({run, functionsDir, timeout, memory}) {
   const files = await readdir(functionsDir);
   for (const file of files) {
     const [actionName, extension] = file.split('.');
     let command =
       `nim action update ${actionName} ${join(functionsDir, file)} ` +
       `--timeout=${Number(timeout)} --memory=${Number(memory)} ` +
-      `--web=raw --env-file=${secretsPath} `;
+      `--web=raw `;
```
Comment (rabbah) [inline index.js:37]:
```suggestion
      `--web=raw`;
```

Comment (rabbah) [reply index.js:37]: does `run.command` re-export `process.env`?

Comment (satyarohith) [reply index.js:37]: Yes. https://github.com/sindresorhus/execa#extendenv

The space is on purpose.

Comment (rabbah) [reply index.js:37]: > The space is on purpose.

K - a comment in the future will be helpful.

**Review by rabbah — APPROVED** [review]

**Pull request merged**